frontend/src/features/contacts/hooks/useAddContact.ts

```diff
@@ -0,0 +1,55 @@
+import axios, { AxiosError } from 'axios';
+import { useApiContacts } from 'hooks/pims-api/useApiContacts';
+import { IApiError } from 'interfaces/IApiError';
+import { ICreatePerson } from 'interfaces/ICreateContact';
+import { useDispatch } from 'react-redux';
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+import { toast } from 'react-toastify';
+import { logError } from 'store/slices/network/networkSlice';
+
+/**
+ * hook that adds a contact.
+ */
+const useAddContact = () => {
+  const { postPerson } = useApiContacts();
+  const dispatch = useDispatch();
+
+  const addPerson = async (
+    person: ICreatePerson,
+    needsUserAction: (needsAction: boolean) => void,
+    userOverride: boolean = false,
+  ) => {
+    try {
+      dispatch(showLoading());
+      const response = await postPerson(person, userOverride);
+      toast.success('Lease/License saved');
```

Comment:
This should probably be more like: "Contact/Person saved" ?

Reply:
Good catch, will update it